.sidekick.toml

```diff
@@ -33,9 +33,9 @@ version = "0.1.0-rc2"
 'package:serde'      = 'force-used=true,package=serde,version=1.0.216,feature=serde_derive'
 'package:serde_with' = 'force-used=true,package=serde_with,version=3.12.0,default-features=false,feature=base64,feature=macros,feature=std'
 # These are used by crates with services.
-'package:async-trait' = 'required-by-services=true,package=async-trait,version=0.1.83'
+'package:async-trait' = 'required-by-services=true,package=async-trait,version=0.1.84'
```

Comment:
Q: were these necessary?

Reply:
No. Kind of. Maybe. I don't know 😵‍💫 

I did a `cargo update` to get all the dependencies updated (since a lot of them get removed / changed by removing `/auth`), and that requires updating the generator configuration.